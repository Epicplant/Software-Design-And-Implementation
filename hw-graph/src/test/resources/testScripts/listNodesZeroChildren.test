# Tests that we correctly have a list of 0 nodes+edges when ListChildren is called
# and the node has 0 children

# Creates our new graph
CreateGraph newGraph

# Adds a new node "n1"
AddNode newGraph n1

# Lists all of the children of this node
ListChildren newGraph n1