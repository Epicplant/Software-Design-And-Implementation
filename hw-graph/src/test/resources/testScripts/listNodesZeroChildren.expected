# Tests that we correctly have a list of 0 nodes+edges when ListChildren is called
# and the node has 0 children

# Creates our new graph
created graph newGraph

# Adds a new node "n1"
added node n1 to newGraph

# Lists all of the children of this node
the children of n1 in newGraph are: